projects/driver_controls/inc/mech_brake_calibration.h

```diff
@@ -0,0 +1,45 @@
+#pragma once
+#include "mech_brake.h"
+// Calibrates the mechanical brake through user-controlled procedure.
+// Requires ADS1015 and soft timers to be initialized.
+//
+// This module is expected to be driven through a calibration FSM.
```

Comment:
You just copy and pasted this from throttle. I doubt this module is expected to be driven through a calibration FSM.